### PR TITLE
Reduce deprecation warnings.

### PR DESCRIPTION
**Pull request by jensens**

### CODE REVIEW

**Comment by davisagli** [comment]
@jensens I'm not sure if you planned to do more here before submitting it? It needs a changelog and green jenkins jobs, at least

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
I am not sure if I catched them all, but lets merge anyway if it goes green.

File: Products/CMFPlone/skins/plone_scripts/queryCatalog.py [hunk 1]
```diff
@@ -7,8 +7,8 @@
 ##parameters=REQUEST=None,show_all=0,quote_logic=0,quote_logic_indexes=['SearchableText','Description','Title'],use_types_blacklist=False,show_inactive=False,use_navigation_root=False
 ##title=wraps the portal_catalog with a rules qualified query
 
+from plone.base.navigationroot import get_navigation_root
```
Comment (davisagli) [inline Products/CMFPlone/skins/plone_scripts/queryCatalog.py:10]: @jensens The import in RestrictedPython is failing. Can we replace this script?

Comment (jensens) [reply Products/CMFPlone/skins/plone_scripts/queryCatalog.py:10]: If we finish #3515 all are gone.

Comment (jensens) [reply Products/CMFPlone/skins/plone_scripts/queryCatalog.py:10]: Lets try in https://github.com/plone/Products.CMFPlone/pull/3960
For grep the word is difficult.

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

File: Products/CMFPlone/skins/plone_scripts/queryCatalog.py (hunk 1 above)
Comment (jensens) [reply Products/CMFPlone/skins/plone_scripts/queryCatalog.py]: this is done and PR is green

**Pull request merged**